pull_alerts.py

```diff
@@ -6,6 +6,8 @@
 import urllib
 from collections import Counter, OrderedDict, defaultdict, namedtuple
 from datetime import datetime, timedelta
+from zoneinfo import ZoneInfo
```

Comment:
This is package is only available for Python 3.9+. Lyft internally still uses 3.8 by default across the board, so this will break many teams usage I imagine.

The Python datetime docs provide an example tzinfo subclass that handles US daylight savings timezones: [here](https://docs.python.org/3.8/library/datetime.html#tzinfo-objects:~:text=)%0A%20%20%20%20return%20dt-,%23%20US%20DST%20Rules,-%23%0A%23%20This%20is)

Alternatively, if you only want a specific timezone _always_, you can hack it with the `TZ` env variable:

```python
>>> import os
>>> import time
>>> from datetime import datetime
>>> print('Default:', datetime.now())
Default: 2023-08-14 10:22:30.101441

>>> for tz in ('UTC', 'America/New_York', 'America/Los_Angeles'):
...     os.environ['TZ'] = tz
...     time.tzset()
...     print(f'{tz}: {datetime.now()}')
...
UTC: 2023-08-14 17:22:30.387957
America/New_York: 2023-08-14 13:22:30.389069
America/Los_Angeles: 2023-08-14 10:22:30.390017
```

Reply:
I thought that since the timezone itself is not important, only its presence is important, then I can just use the LOCAL_TZ variable here without even changing it, what do you think?

Reply:
Oh sure. That looks like it might work.

Overall though I think it should be a command line argument you can pass to use a specific timezone. I don't think it should be hard-coded to always use pacific time.

Reply:
Could your immediate problem be just setting the `TZ` environment variable yourself? Like so:

```shell
TZ='America/Los_Angeles' pull_alerts.py
```

This way no code changes would even be necessary

Reply:
Yeah, I can solve it locally, but then this problem will remain for everyone who uses this repository from Europe timezone